cactusbot/cached/caching.py

```diff
@@ -0,0 +1,18 @@
+class CacheUtils(object):
+
+    # You have found MysticalMage's code, turn back now, no good lies ahead
+    def in_cache_followers(user):
+        """ Returns if user is in the cache. """
+        follower_cache = open("caches/CacheFollowers.txt", "r")
+        for line in follower_cache:
+            if user in line:
+                follower_cache.close()
+                return True
+        follower_cache.close()
+        return False
+
+    def cache_followers_add(user):
+        """ Adds user to cache with newline at end. """
+        follower_cache = open("caches/CacheFollowers.txt", "a+")
+        follower_cache.write(user + "\n")
```

Comment:
We should cache users with a timestamp. That way, if they follow, unfollow, and refollow six months later, they get another notification.

```json
{
  "user1": "2016-10-10 01:28:46.919407",
  "user2": "2016-8-22 23:05:47.329743"
}
```
```python
if username in cache:
    return cache["username"]
```